blosc/blosc.c

```diff
@@ -90,10 +90,8 @@ struct blosc_context {
 
   const uint8_t* src;
   uint8_t* dest;                  /* The current pos in the destination buffer */
-  uint8_t* header_flags;          /* Flags for header.  Currently booked:
-                                    - 0: byte-shuffled?
-                                    - 1: memcpy'ed?
-                                    - 2: bit-shuffled? */
+  uint8_t* header_flags;          /* Flags for header */
+  int versionlz;                  /* Compressor version byte, only used during decompression */
```

Comment:
How about calling it just `compressor_version`? (It looks like it's used for more than just LZ-based compressors?)